### ISSUE TITLE
Question : How to change the default position of a window : screen_x screen_y.

### ISSUE THREAD

**Issue by vvw2wvv-live** [hecrj/iced#556]
and how to get the screen size : screen_width screen_height

and and can we create a frameless window with iced?

thanks a lot

**Comment by twitchyliquid64** [comment]
> How to change the default position of a window

Iced doesnt have a way to do that ATM - but, there exists `set_outer_position` in the [underlying library](https://docs.rs/winit/0.23.0/src/winit/window.rs.html#465-467), so potentially a simple code change could implement it?

> how to get the screen size 

Im not sure how to get the current screen size, but if you are making a widget, its best to use the information provided through the `layout()` method.

You can also try looking for window resize events, using a technique like this: https://github.com/hecrj/iced/tree/master/examples/events

> and and can we create a frameless window with iced?

Set `iced_winit::Window`.decorations to `false`.

**Comment by qq389146213** [comment]
set_outer_position has a parameter
 position: Into<Position>
 how to pass it in with Settings::window::Settings

**Comment by hecrj** [comment]
This is a duplicate of #515.

We are waiting on https://github.com/rust-windowing/winit/issues/1190.
